openff/toolkit/tests/test_toolkits.py

```diff
@@ -2834,7 +2859,32 @@ def test_assign_partial_charges_am1bcc(self):
         for pc in molecule._partial_charges:
             charge_sum += pc
             abs_charge_sum += abs(pc)
-        assert abs(charge_sum) < 0.001 * unit.elementary_charge
+            assert abs(charge_sum) < 1e-10 * unit.elementary_charge
+            assert abs_charge_sum > 0.25 * unit.elementary_charge
+
+    def test_assign_partial_charges_am1bcc_no_normalization(self):
+        """Test AmberToolsToolkitWrapper assign_partial_charges() with am1bcc, with
+        normalize_partial_charges=False"""
+        toolkit_registry = ToolkitRegistry(
+            toolkit_precedence=[AmberToolsToolkitWrapper, RDKitToolkitWrapper]
+        )
+        # Use a cyclic N3H3 molecule since this is likely to produce a rounding error
+        # Antechamber outputs 6 digits after the decimal in charges.txt, so I (Jeff) don't know
+        # why this N3H3 molecule ends up with an error of 1e-3, but it's the smallest reproducing
+        # case of this that I could find.
+        molecule = create_cyclic_n3h3()
+        molecule.assign_partial_charges(
+            partial_charge_method="am1bcc",
+            toolkit_registry=toolkit_registry,
+            normalize_partial_charges=False,
+        )
+        charge_sum = 0 * unit.elementary_charge
+        abs_charge_sum = 0 * unit.elementary_charge
+        for pc in molecule._partial_charges:
+            charge_sum += pc
+            abs_charge_sum += abs(pc)
```

Comment:
Are `charge_sum` and `abs_charge_sum` expected to be the same for both toolkits? That is, could the test here us a known good number rather than computing it? I ask because as testing philosophy, this test seems to verify that two implementations of the same algorithm, when run twice, give the same answer. I usually like to have known "golden" value to compare against, or a different algorithm, with a strong preference for the former.

Reply:
For methods that require 3D geometries, like AM1(-BCC) and PM3, we can't expect the same charges from all providers. This is ultimately due to deep implementation details (Both OE and AmberTools AM1 calculations do a geometry optimization of the input conformers, but with different settings, so they can reasonably provide two different-yet-correct sets of charges). 

On the other hand, for graph-based charge methods (like MMFF94 and Gasteiger), we could do a strict test for "gold standard" outputs. I'd consider that outside the scope of this PR, and somewhat low priority, since 99% of our users will be using AM1.